git/install_hooks.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+curl -H "Authorization: token $GH_ACCESS_TOKEN" \
+     -o .git/hooks/prepare-commit-msg \
+     --create-dirs \
+     https://raw.githubusercontent.com/banksalad/styleguide/git/hooks/prepare-commit-msg
+chmod +x .git/hooks/prepare-commit-msg
```

Comment:
```suggestion
chmod +x .git/hooks/prepare-commit-msg

```
P2; 개행열 챙겨야할 것 같습니다.